packages/material/scss/popup/_layout.scss

```diff
@@ -3,6 +3,8 @@ $popup-header-padding: $padding-y-lg $padding-x-lg !default;
 $popup-header-font-size: $font-size-sm !default;
 $popup-header-text: rgba($base-text, .54) !default;
 $popup-filter-padding-x: $padding-x;
+$dropdowntree-popup-spacer-y: 0 !default;
+$dropdowntree-popup-spacer-x: $popup-padding-y !default;
```

Comment:
These variables are for the dropdowntree, and they should not be in the popup files.